Cargo.toml

```diff
@@ -11,7 +11,7 @@ exclude = ["Gir*.toml", "tests/**/*", "*.md"]
 
 [dependencies]
 docopt = "*"
-xml-rs = "0.2.1"
+xml-rs = "*"
```

Comment:
I'm clearly not a big fan of this. Also, we won't be able to publish we use '*' (even if we don't care).